airflow/www/security.py

```diff
@@ -235,6 +235,71 @@ def add_permissions(self, role, perms):
             perm_view = self.add_permission_view_menu(perm_name, view_name)
             self.add_permission_role(role, perm_view)
 
+    def get_resource(self, name: str) -> ViewMenu:
+        """
+        Returns a resource record by name, if it exists.
+        :param name: Name of resource
```

Comment:
Sphinx needs one blank line between description and params

```suggestion
        Returns a resource record by name, if it exists.

        :param name: Name of resource
```